sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ClassNotInstantiatable.cs

```diff
@@ -94,4 +95,13 @@ private WithAttribute2()
         {
         }
     }
+
+    // https://github.com/SonarSource/sonar-dotnet/issues/3329
+    public class Repro_3329 : System.Runtime.InteropServices.SafeHandle
```

Comment:
Preserve the original comment here please to explain why is this case compliant (except the Non+FP part)
```suggestion
    public class Repro_3329 : System.Runtime.InteropServices.SafeHandle 'Compliant, instance will be created by PInvoke of DllImport
```